### ISSUE TITLE
Background sync for locks

### ISSUE THREAD

**Issue by Tusky** [SRombauts/UE4GitPlugin#84]
Would be great if there were an option, to background sync which would ping the locked file list, and update the UI in Unreal if it found new locks.

**Comment by NazzarenoGiannelli** [comment]
Yes Please!
This would mean A LOT! :)

**Comment by SRombauts** [comment]
Hi, just to be sure :
I think Unreal is already doing some refresh when you change directory (after some cool down) in the content browser ; are you seeing those correctly?
The thing is you might want, either to get some update without any interaction with the content browser, or you might want smaller time intervals (cool downs) 
These are currently Editor side

**Comment by NazzarenoGiannelli** [comment]
Like Perforce is already doing, it would be nice to have Source Control icons showing if someone is working on some asset. You browse the content browser and whenever you stumble upon a folder where some file is being worked, you see the relative icons, without the need or syncing or hoping that no one else is messing with the same stuff you want.

Right now with your plugin if I do not use LFS 2 Locking system everything works fine: I see what I have added and checked out without seeing others' stuff. If I enable the LFS 2 Locking system I only get blue checkmarks even if I checked out the asset.

Probably it's a confusing explanation. If needed I could explain better, maybe with a couple of screenshots,

**Comment by Tusky** [comment]
Yes. what I meant was if I start messing around with a file that's great it shows everything, but if a team member does it, unless I sync no matter how much time passes, I never see that the file is locked. I could just try and edit it, and once I would push/commit (been awhile don't remember which exactly triggered it) it, *then* it would tell me that hey, that file is locked.

**Comment by martenzander** [comment]
Yes, having the yellow checkmark just like with Perforce, would be very nice
